fcrepo-kernel-api/src/main/java/org/fcrepo/kernel/api/models/FedoraResource.java

```diff
@@ -257,4 +257,10 @@ void replaceProperties(final IdentifierConverter<Resource, FedoraResource> idTra
      * @return the resource being described
      */
     FedoraResource getDescribedResource();
+
+    /**
+     * Get a graph resource corresponding to this resource
+     * @return a graph resource
+     */
+    Resource graphResource(Converter<Resource, String> idTranslator);
```

Comment:
I'll suggest going ahead and calling this a URI. It's going to be a URI, and that's what people want.